tools/dotcms-cli/cli/src/main/java/com/dotcms/cli/command/language/LanguagePush.java

```diff
@@ -0,0 +1,118 @@
+package com.dotcms.cli.command.language;
+
+import com.dotcms.api.LanguageAPI;
+import com.dotcms.api.client.RestClientFactory;
+import com.dotcms.cli.common.HelpOption;
+import com.dotcms.cli.common.OutputOptionMixin;
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.model.ResponseEntityView;
+import com.dotcms.model.language.Language;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.File;
+import java.io.IOException;
+import java.util.Optional;
+import java.util.concurrent.Callable;
+import javax.enterprise.context.control.ActivateRequestContext;
+import javax.inject.Inject;
+import org.apache.commons.lang3.StringUtils;
+import picocli.CommandLine;
+
+@ActivateRequestContext
+@CommandLine.Command(
+        name = LanguagePush.NAME,
+        description = "@|bold,green Save or update a language given a Language object (in JSON or YML format) or tag (e.g.: en-us)|@"
+)
+/**
+ * Command to push a language given a Language object (in JSON or YML format) or tag (e.g.: en-us)
+ * @author nollymar
+ */
+public class LanguagePush extends AbstractLanguageCommand implements Callable<Integer> {
+    static final String NAME = "push";
+
+    @CommandLine.Mixin(name = "output")
+    OutputOptionMixin output;
+
+    @CommandLine.Mixin
+    protected HelpOption helpOption;
+
+    @CommandLine.Option(names = {"--byTag"}, description = "Tag to be used to create a new language. Used when no file is specified. For example: en-us")
+    String languageTag;
+
+    @CommandLine.Option(names = {"-f", "--file"}, description = "The json/yml formatted content-type descriptor file to be pushed. ")
```

Comment:
I think All commands are expected to have a "default action" so to speak 
I mean We should be able to say `language push ~/cr-es.json   ` without having to specify the -f option 
Or at least that is what I was told about the other command

Reply:
the thing is that for languages the file is optional because I can push languages by tag with the `--byTag` flag or by file. Is there a way to do it without the `-f`?